docs/rate-limits.rst

```diff
@@ -33,11 +33,11 @@ the API.  When several rates are given, all are enforced at the same time.
 +-----------------------------------------+----------+-------------------------------------------------------------------------------------------+
 | ``dyndns``                              | 1/min    | dynDNS updates (per domain).                                                              |
 +-----------------------------------------+----------+-------------------------------------------------------------------------------------------+
-| ``dns_api_cheap``                       | 10/s     | most DNS read operations (e.g. fetching an RRset)                                         |
+| ``dns_api_cheap``                       | 10/s     | DNS read operations (e.g. fetching an RRset)                                              |
```

Comment:
I understand the zonefile export as a 'read' operation, hence the change.

Reply:
yes, but it was in the wrong commit. The commit that adds the functionality now changes this line to say "except zonefile export" (see other fixup)